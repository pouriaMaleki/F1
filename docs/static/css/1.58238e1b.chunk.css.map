{"version":3,"sources":["/Users/pouriamaleki/Projects/Personal/f1/node_modules/roboto-fontface/css/roboto/roboto-fontface.css"],"names":[],"mappings":"AAAA,WACI,mBAAsB,AACtB,8HAA4H,AAC5H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,wBAA2B,AAC3B,6HAA4H,CAC/H,AAED,WACI,mBAAsB,AACtB,0IAAwI,AACxI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,8BAAiC,AACjC,yIAAwI,CAC3I,AAED,WACI,mBAAsB,AACtB,gIAA8H,AAC9H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,yBAA4B,AAC5B,+HAA8H,CACjI,AAED,WACI,mBAAsB,AACtB,4IAA0I,AAC1I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,+BAAkC,AAClC,2IAA0I,CAC7I,AAED,WACI,mBAAsB,AACtB,oIAAkI,AAClI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,2BAA8B,AAC9B,mIAAkI,CACrI,AAED,WACI,mBAAsB,AACtB,gJAA8I,AAC9I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,iCAAoC,AACpC,+IAA8I,CACjJ,AAED,WACI,mBAAsB,AACtB,kIAAgI,AAChI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,0BAA6B,AAC7B,iIAAgI,CACnI,AAED,WACI,mBAAsB,AACtB,8IAA4I,AAC5I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,gCAAmC,AACnC,6IAA4I,CAC/I,AAED,WACI,mBAAsB,AACtB,8HAA4H,AAC5H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,wBAA2B,AAC3B,6HAA4H,CAC/H,AAED,WACI,mBAAsB,AACtB,0IAAwI,AACxI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,8BAAiC,AACjC,yIAAwI,CAC3I,AAED,WACI,mBAAsB,AACtB,gIAA8H,AAC9H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,yBAA4B,AAC5B,+HAA8H,CACjI,AAED,WACI,mBAAsB,AACtB,4IAA0I,AAC1I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,+BAAkC,AAClC,2IAA0I,CAC7I","file":"1.58238e1b.chunk.css","sourcesContent":["@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Thin.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Thin.woff') format('woff');\n    font-weight: 100;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Thin';\n    src: url('../../fonts/roboto/Roboto-Thin.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Thin.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-ThinItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-ThinItalic.woff') format('woff');\n    font-weight: 100;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-ThinItalic';\n    src: url('../../fonts/roboto/Roboto-ThinItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-ThinItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Light.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Light.woff') format('woff');\n    font-weight: 300;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Light';\n    src: url('../../fonts/roboto/Roboto-Light.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Light.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-LightItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-LightItalic.woff') format('woff');\n    font-weight: 300;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-LightItalic';\n    src: url('../../fonts/roboto/Roboto-LightItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-LightItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Regular.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Regular.woff') format('woff');\n    font-weight: 400;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Regular';\n    src: url('../../fonts/roboto/Roboto-Regular.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Regular.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-RegularItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-RegularItalic.woff') format('woff');\n    font-weight: 400;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-RegularItalic';\n    src: url('../../fonts/roboto/Roboto-RegularItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-RegularItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Medium.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Medium.woff') format('woff');\n    font-weight: 500;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Medium';\n    src: url('../../fonts/roboto/Roboto-Medium.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Medium.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-MediumItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-MediumItalic.woff') format('woff');\n    font-weight: 500;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-MediumItalic';\n    src: url('../../fonts/roboto/Roboto-MediumItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-MediumItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Bold.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Bold.woff') format('woff');\n    font-weight: 700;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Bold';\n    src: url('../../fonts/roboto/Roboto-Bold.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Bold.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-BoldItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BoldItalic.woff') format('woff');\n    font-weight: 700;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-BoldItalic';\n    src: url('../../fonts/roboto/Roboto-BoldItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BoldItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Black.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Black.woff') format('woff');\n    font-weight: 900;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Black';\n    src: url('../../fonts/roboto/Roboto-Black.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Black.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-BlackItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BlackItalic.woff') format('woff');\n    font-weight: 900;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-BlackItalic';\n    src: url('../../fonts/roboto/Roboto-BlackItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BlackItalic.woff') format('woff');\n}\n"]}