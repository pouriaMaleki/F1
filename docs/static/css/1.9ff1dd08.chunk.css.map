{"version":3,"sources":["/Users/pouriamaleki/Projects/Personal/f1/node_modules/roboto-fontface/css/roboto/roboto-fontface.css"],"names":[],"mappings":"AAAA,WACI,mBAAsB,AACtB,kIAA4H,AAC5H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,wBAA2B,AAC3B,iIAA4H,CAC/H,AAED,WACI,mBAAsB,AACtB,8IAAwI,AACxI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,8BAAiC,AACjC,6IAAwI,CAC3I,AAED,WACI,mBAAsB,AACtB,oIAA8H,AAC9H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,yBAA4B,AAC5B,mIAA8H,CACjI,AAED,WACI,mBAAsB,AACtB,gJAA0I,AAC1I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,+BAAkC,AAClC,+IAA0I,CAC7I,AAED,WACI,mBAAsB,AACtB,wIAAkI,AAClI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,2BAA8B,AAC9B,uIAAkI,CACrI,AAED,WACI,mBAAsB,AACtB,oJAA8I,AAC9I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,iCAAoC,AACpC,mJAA8I,CACjJ,AAED,WACI,mBAAsB,AACtB,sIAAgI,AAChI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,0BAA6B,AAC7B,qIAAgI,CACnI,AAED,WACI,mBAAsB,AACtB,kJAA4I,AAC5I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,gCAAmC,AACnC,iJAA4I,CAC/I,AAED,WACI,mBAAsB,AACtB,kIAA4H,AAC5H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,wBAA2B,AAC3B,iIAA4H,CAC/H,AAED,WACI,mBAAsB,AACtB,8IAAwI,AACxI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,8BAAiC,AACjC,6IAAwI,CAC3I,AAED,WACI,mBAAsB,AACtB,oIAA8H,AAC9H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,yBAA4B,AAC5B,mIAA8H,CACjI,AAED,WACI,mBAAsB,AACtB,gJAA0I,AAC1I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,+BAAkC,AAClC,+IAA0I,CAC7I","file":"1.9ff1dd08.chunk.css","sourcesContent":["@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Thin.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Thin.woff') format('woff');\n    font-weight: 100;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Thin';\n    src: url('../../fonts/roboto/Roboto-Thin.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Thin.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-ThinItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-ThinItalic.woff') format('woff');\n    font-weight: 100;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-ThinItalic';\n    src: url('../../fonts/roboto/Roboto-ThinItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-ThinItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Light.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Light.woff') format('woff');\n    font-weight: 300;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Light';\n    src: url('../../fonts/roboto/Roboto-Light.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Light.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-LightItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-LightItalic.woff') format('woff');\n    font-weight: 300;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-LightItalic';\n    src: url('../../fonts/roboto/Roboto-LightItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-LightItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Regular.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Regular.woff') format('woff');\n    font-weight: 400;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Regular';\n    src: url('../../fonts/roboto/Roboto-Regular.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Regular.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-RegularItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-RegularItalic.woff') format('woff');\n    font-weight: 400;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-RegularItalic';\n    src: url('../../fonts/roboto/Roboto-RegularItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-RegularItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Medium.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Medium.woff') format('woff');\n    font-weight: 500;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Medium';\n    src: url('../../fonts/roboto/Roboto-Medium.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Medium.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-MediumItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-MediumItalic.woff') format('woff');\n    font-weight: 500;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-MediumItalic';\n    src: url('../../fonts/roboto/Roboto-MediumItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-MediumItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Bold.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Bold.woff') format('woff');\n    font-weight: 700;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Bold';\n    src: url('../../fonts/roboto/Roboto-Bold.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Bold.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-BoldItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BoldItalic.woff') format('woff');\n    font-weight: 700;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-BoldItalic';\n    src: url('../../fonts/roboto/Roboto-BoldItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BoldItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Black.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Black.woff') format('woff');\n    font-weight: 900;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Black';\n    src: url('../../fonts/roboto/Roboto-Black.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Black.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-BlackItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BlackItalic.woff') format('woff');\n    font-weight: 900;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-BlackItalic';\n    src: url('../../fonts/roboto/Roboto-BlackItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BlackItalic.woff') format('woff');\n}\n"]}