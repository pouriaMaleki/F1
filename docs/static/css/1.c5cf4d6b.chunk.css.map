{"version":3,"sources":["/Users/pouriamaleki/Projects/Personal/f1/node_modules/roboto-fontface/css/roboto/roboto-fontface.css"],"names":[],"mappings":"AAAA,WACI,mBAAsB,AACtB,wIAA4H,AAC5H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,wBAA2B,AAC3B,uIAA4H,CAC/H,AAED,WACI,mBAAsB,AACtB,oJAAwI,AACxI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,8BAAiC,AACjC,mJAAwI,CAC3I,AAED,WACI,mBAAsB,AACtB,0IAA8H,AAC9H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,yBAA4B,AAC5B,yIAA8H,CACjI,AAED,WACI,mBAAsB,AACtB,sJAA0I,AAC1I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,+BAAkC,AAClC,qJAA0I,CAC7I,AAED,WACI,mBAAsB,AACtB,8IAAkI,AAClI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,2BAA8B,AAC9B,6IAAkI,CACrI,AAED,WACI,mBAAsB,AACtB,0JAA8I,AAC9I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,iCAAoC,AACpC,yJAA8I,CACjJ,AAED,WACI,mBAAsB,AACtB,4IAAgI,AAChI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,0BAA6B,AAC7B,2IAAgI,CACnI,AAED,WACI,mBAAsB,AACtB,wJAA4I,AAC5I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,gCAAmC,AACnC,uJAA4I,CAC/I,AAED,WACI,mBAAsB,AACtB,wIAA4H,AAC5H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,wBAA2B,AAC3B,uIAA4H,CAC/H,AAED,WACI,mBAAsB,AACtB,oJAAwI,AACxI,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,8BAAiC,AACjC,mJAAwI,CAC3I,AAED,WACI,mBAAsB,AACtB,0IAA8H,AAC9H,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,yBAA4B,AAC5B,yIAA8H,CACjI,AAED,WACI,mBAAsB,AACtB,sJAA0I,AAC1I,gBAAiB,AACjB,iBAAmB,CACtB,AAED,WACI,+BAAkC,AAClC,qJAA0I,CAC7I","file":"1.c5cf4d6b.chunk.css","sourcesContent":["@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Thin.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Thin.woff') format('woff');\n    font-weight: 100;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Thin';\n    src: url('../../fonts/roboto/Roboto-Thin.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Thin.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-ThinItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-ThinItalic.woff') format('woff');\n    font-weight: 100;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-ThinItalic';\n    src: url('../../fonts/roboto/Roboto-ThinItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-ThinItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Light.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Light.woff') format('woff');\n    font-weight: 300;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Light';\n    src: url('../../fonts/roboto/Roboto-Light.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Light.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-LightItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-LightItalic.woff') format('woff');\n    font-weight: 300;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-LightItalic';\n    src: url('../../fonts/roboto/Roboto-LightItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-LightItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Regular.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Regular.woff') format('woff');\n    font-weight: 400;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Regular';\n    src: url('../../fonts/roboto/Roboto-Regular.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Regular.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-RegularItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-RegularItalic.woff') format('woff');\n    font-weight: 400;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-RegularItalic';\n    src: url('../../fonts/roboto/Roboto-RegularItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-RegularItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Medium.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Medium.woff') format('woff');\n    font-weight: 500;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Medium';\n    src: url('../../fonts/roboto/Roboto-Medium.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Medium.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-MediumItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-MediumItalic.woff') format('woff');\n    font-weight: 500;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-MediumItalic';\n    src: url('../../fonts/roboto/Roboto-MediumItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-MediumItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Bold.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Bold.woff') format('woff');\n    font-weight: 700;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Bold';\n    src: url('../../fonts/roboto/Roboto-Bold.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Bold.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-BoldItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BoldItalic.woff') format('woff');\n    font-weight: 700;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-BoldItalic';\n    src: url('../../fonts/roboto/Roboto-BoldItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BoldItalic.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-Black.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Black.woff') format('woff');\n    font-weight: 900;\n    font-style: normal;\n}\n\n@font-face {\n    font-family: 'Roboto-Black';\n    src: url('../../fonts/roboto/Roboto-Black.woff2') format('woff2'), url('../../fonts/roboto/Roboto-Black.woff') format('woff');\n}\n\n@font-face {\n    font-family: 'Roboto';\n    src: url('../../fonts/roboto/Roboto-BlackItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BlackItalic.woff') format('woff');\n    font-weight: 900;\n    font-style: italic;\n}\n\n@font-face {\n    font-family: 'Roboto-BlackItalic';\n    src: url('../../fonts/roboto/Roboto-BlackItalic.woff2') format('woff2'), url('../../fonts/roboto/Roboto-BlackItalic.woff') format('woff');\n}\n"]}