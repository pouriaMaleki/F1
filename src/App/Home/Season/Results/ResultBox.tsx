import React from "react";
import css from "./ResultBox.module.sass";
import classNames from "classnames";
import { IResult } from "../IResult";

interface IProps extends IResult {
  className?: string;
}

const ResultBox = (props: IProps) => (
  <div className={classNames(css.root, props.className)}>
    <div className={css.title}>{props.title}</div>
    <div className={css.value}>{props.value}</div>
  </div>
);

export { ResultBox };
