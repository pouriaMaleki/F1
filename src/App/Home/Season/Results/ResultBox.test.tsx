import React from "react";
import renderer from "react-test-renderer";
import { mount } from "enzyme";
import { ResultBox } from "./ResultBox";
import css from "./ResultBox.module.sass";

describe("Render ResultBox", () => {
  // snapshot test
  it("render correctly ResultBox component", () => {
    const ResultBoxComponent = renderer
      .create(<ResultBox title="title" value="value" />)
      .toJSON();
    expect(ResultBoxComponent).toMatchSnapshot();
  });

  // test props
  it("add className as props", () => {
    const CLASSNAME = "CLASSNAME";
    const props = {
        className: CLASSNAME
      },
      ResultBoxComponent = mount(
        <ResultBox title="title" value="value" {...props} />
      );

    expect(ResultBoxComponent.hasClass(CLASSNAME)).toEqual(true);
  });

  it("render proper title", () => {
    const props = {
        title: "title",
        value: "value"
      },
      ResultBoxComponent = mount(<ResultBox {...props} />);

    expect(ResultBoxComponent.find("." + css.title).text()).toEqual(
      props.title
    );
  });

  it("render proper value", () => {
    const props = {
        title: "title",
        value: "value"
      },
      ResultBoxComponent = mount(<ResultBox {...props} />);

    expect(ResultBoxComponent.find("." + css.value).text()).toEqual(
      props.value
    );
  });
});
