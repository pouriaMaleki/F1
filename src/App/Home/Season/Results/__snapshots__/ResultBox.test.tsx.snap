// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ResultBox render correctly ResultBox component 1`] = `
<div
  className="root"
>
  <div
    className="title"
  >
    title
  </div>
  <div
    className="value"
  >
    value
  </div>
</div>
`;
